.github/workflows/ci.yaml

```diff
@@ -65,6 +66,7 @@ jobs:
           path: /tmp/gradle_spotbugs_reports
 
   dockerfile-lint:
+    name: Lint dockerfiles
```

Comment:
I added a more human-readable name for each job to improve the readability of job names when checking the web UI.